server/src/main/java/org/elasticsearch/action/admin/cluster/stats/RemoteClusterStatsRequest.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+package org.elasticsearch.action.admin.cluster.stats;
+
+import org.elasticsearch.TransportVersions;
+import org.elasticsearch.action.ActionRequest;
+import org.elasticsearch.action.ActionRequestValidationException;
+import org.elasticsearch.common.io.stream.StreamInput;
+import org.elasticsearch.common.io.stream.StreamOutput;
+
+import java.io.IOException;
+
+/**
+ * A request to get cluster level stats from the remote cluster.
+ */
+public class RemoteClusterStatsRequest extends ActionRequest {
+    public RemoteClusterStatsRequest(StreamInput in) {
+        this();
+    }
+
+    public RemoteClusterStatsRequest() {
+        super();
+    }
+
+    @Override
+    public ActionRequestValidationException validate() {
+        return null;
+    }
+
+    @Override
+    public void writeTo(StreamOutput out) throws IOException {
+        if (out.getTransportVersion().before(TransportVersions.CCS_TELEMETRY_STATS)) {
```

Comment:
This transport version was already added in #112310 and does not imply that the remote node knows what we're talking about here. We need a new transport version just for this functionality.

Reply:
We do? I thought since we're under a feature flag there's no way we can encounter a mix  between https://github.com/elastic/elasticsearch/pull/112310 and this, but I can add it if it's necessary.

Reply:
Feature flags don't have a good rolling-upgrade story. I think we should forget about it entirely and just make this predicated on the version.

Reply:
OK I'll drop the flag from here.